pronto_quadruped_ros/src/legodo_handler_ros.cpp

```diff
@@ -72,11 +73,19 @@ LegodoHandlerBase::LegodoHandlerBase(ros::NodeHandle &nh,
     // }
     // imu_sub_ = nh_.subscribe(imu_topic_, 100, &LegodoHandlerROS::imuCallback, this);
 
-    std::vector<std::string> leg_names = {"lf", "rf", "lh", "rh"};
+    nh.param<std::string>("base_link_name", base_link_name_, "base");
+    foot_names_.resize(4);
+    nh.param<std::string>("LF_FOOT_name", foot_names_[0], "LF_FOOT");
```

Comment:
better print this after it gets fetched.

Why not having this as a vector of strings in the yaml?
i.e. 
```
foot_names : [ "LF_FOOT", "RF_FOOT", "LH_FOOT", "RH_FOOT" ]
```

Reply:
I did not want to force an implicit order we'd need to document (and I am using the same parameter in a second place). If you prefer the vector, let me know and I'll amend.

Reply:
I think we should use a vector to avoid clutter. The ordering is and will always be LF, RF, LH, RH 
DId SOLO or other robots use a different convention?